src/components/GameDetail/index.jsx

```diff
@@ -21,74 +22,63 @@ export default function GameDetail() {
     website,
   } = gameData;
 
-  const options = {
-    method: "GET",
-    url: `https://rawg-video-games-database.p.rapidapi.com/games/${gameId}`,
-    params: { key: import.meta.env.VITE_RAWG_KEY },
-    headers: {
-      "X-RapidAPI-Key": import.meta.env.VITE_RAPID_API_KEY,
-      "X-RapidAPI-Host": "rawg-video-games-database.p.rapidapi.com",
-    },
-  };
-
   useEffect(() => {
-    (async function () {
-      try {
-        const response = await axios.request(options);
-        console.log(response.data);
+    fetchGames(`games/${gameId}`)
+      .then((response) => {
         setGameData(response.data);
-      } catch (error) {
-        console.log(error);
-      }
-    })();
+      })
+      .catch((err) => console.log(err));
   }, []);
+
   return (
-      <div className="game-detail">
-        <h1 className="game-title">{name || <Skeleton width="400px" />}</h1>
-        <div className="game-detail-container">
-          <div className="game-detail-left">
-            {background_image ? (
-              <img className="game-image" src={background_image} alt="game" />
-            ) : (
-              <Skeleton height="400px" width="53vw" />
-            )}
-            <div className="game-description">
-              <h2>About this Game</h2>
-              <div>
-                {description ? (
-                  parse(`${description}`)
-                ) : (
-                  <Skeleton height="200px" />
-                )}
-              </div>
-            </div>
-          </div>
-          <div className="game-detail-right">
-            <div>
-              <p className="subtitle">Genre: </p>
-              <p className="subtitle-data">
-                {genres ? genres[0].name : <Skeleton />}
-              </p>
-            </div>
-            <div>
-              <p className="subtitle">Developers: </p>
-              <p className="subtitle-data">
-                {developers ? developers[0].name : <Skeleton />}
-              </p>
-            </div>
+    <div className="game-detail">
+      <h1 className="game-title">{name || <Skeleton width="400px" />}</h1>
+      <div className="game-detail-container">
+        <div className="game-detail-left">
+          {background_image ? (
```

Comment:
again mentioning use loading state instead of properties to show loading

Reply:
used loading state yesterday only and its coming in next pull request.

Reply:
![Screenshot 2022-12-06 at 11 04 11](https://user-images.githubusercontent.com/114575114/205825196-e54d4292-de49-447d-a974-7a146eea219f.png)